src/ducks/balance/helpers.js

```diff
@@ -9,7 +9,11 @@ import {
   format as formatDate
 } from 'date-fns'
 
-export const getBalanceHistories = (accounts, transactions) => {
+export const getBalanceHistories = (
+  accounts,
+  transactions,
+  from = startOfToday()
```

Comment:
The startOfToday will only be executed once, when the function is parsed so if you ahve a long running application, over several days (which I think will not be uncommon), we won't have the real start of today.

```
if (!from) { from = startOfToday() }
```

is better in this case.

Reply:
Actually I removed this default parameter later.